### ISSUE TITLE
Help needed in Arctos - list of CITES taxa

### ISSUE THREAD

**Issue by campmlc** [ArctosDB/arctos#8348]
# Tell us what you are trying to do 
I need information on whether a list of taxa are CITES listed to determine fit for use for a loan and for reporting purposes. I know we have discussed this previously, and at one point we had a means of downloading listed taxa. Is this still possible? @ccicero @mkoo

**Comment by ccicero** [comment]
I don't recall that we had a way of downloading listed taxa, although that would be useful! The Arctos Legal classification has what you're looking for but you have to check species one at a time.

@dustymc ???

**Comment by ccicero** [comment]
@dustymc Following up on today's AWG/Issues meeting re: the usage restrictions, what about showing a flag for CITES-listed species in the specimen record where it shows taxonomic classification under identification, It could also potentially show in specimen results next to the name under "identified as" - ???

![image](https://github.com/user-attachments/assets/2495efc8-187e-45ae-9d39-8460e23d365e)

**Comment by dustymc** [comment]
@ccicero I don't think that could be an appropriate assertion from my position - CITES (&etc.) isn't that simple, Arctos could not justify automagically adding some scary and inappropriate warning to [antique ivory napkin rings](https://arctos.database.museum/guid/UAM:EH:UA68-033-0007AB). (@AJLinn I added a taxon to that identification - formerly unidentifiable - please feel free to tell me to knock it off!)

If you want that level of information in your collection, you could add it to whatever [source](https://arctos.database.museum/info/ctDocumentation.cfm?table=cttaxonomy_source) you prefer.

If you wanted something a bit more sophisticated (eg that considers dates and places and all the other stuff that regulations  involve) then you could periodically add records to a permitted (eg noting use restrictions) data loan (and this is a great example of why that structure could use a better name, not that anyone needed another), which would also make those (now reviewed/curated) data available in things like loan UIs (https://github.com/ArctosDB/dev/issues/122).

**Comment by campmlc** [comment]
The challenge of adding records periodically to a restricted use data loan is that we don't know in advance what records are restricted unless we periodically go search taxa one by one through the CITES list. If we have to do that, we have no need for the data loan. In addition, adding data loans needlessly adds to already complex workflows. 
So the original question was - how do we do a search on taxa or records in Arctos that are CITES listed? Do we need to import the CITES list as a taxonomy source?

**Comment by ccicero** [comment]
Right now I think it just shows in the taxonomy classification under Arctos Legal. Could we add a search term to https://arctos.database.museum/taxonomy.cfm specifically for Arctos Legal?

**Comment by dustymc** [comment]
The need/goal is elusive, but "search on .. records in Arctos that are CITES listed?" can (to the extent the classification data are trustworthy) be done with something like [this](https://arctos.database.museum/search.cfm?guid_prefix=MVZ%3ABird&tax_trm_1=%3DI&tax_rnk_1=CITES%20Appendix&tax_src_1=Arctos%20Legal). (See also previous [Issues](https://github.com/search?q=org%3AArctosDB+cites&type=issues) and [discussions](https://github.com/search?q=org%3AArctosDB+cites&type=discussions).)

**Comment by mkoo** [comment]
I remember the discussions but not the outcome-- cool! I think we could definitely come up with Documentation in the Handbook (maybe even some quick link in the Tools Directory? or I dont know where) But basically taxonomy is the place to handle this. So maybe just adding an additional Taxon Identification from Arctos Legal? (could always be 2 so it's secondary to the main id)

Also, is there some metadata with the taxonomy Arctos legal so we know the last update? (https://arctos.database.museum/taxonomy.cfm?taxon_name=&taxon_name_type=&taxon_term=&term_type=&source=Arctos+Legal&creator=#taxonsearchresults) this is basic ignorance on my part. I just downloaded the entire CITES index as CSV

**Comment by campmlc** [comment]
I just searched on MSB:Bird and only got 145 records, mostly US, Canada, Mexico but a few zoo animals with no geography. None of the taxa from my current export show up - these are taxa from South America, Africa, South Pacific, Australia. Is our source limited to just North America?
For example, this one is not found in the search yet it is CITES II: CALLIPHLOX AMETHYSTINA

**Comment by dustymc** [comment]
> adding an additional Taxon Identification

There's no need for that (but the match is by taxon so there's some chance of homonyms doing weird things in the recipe I provided, I don't think this is ever going to be fully magic).

>from Arctos Legal?

and no way to do that

>last update

The data came from a spotty API (in Kenya, I doubt it was NA-centric) that was turned off (because it only ever kinda-maybe worked) - hence the caveat above, I'd be pretty surprised if this was ever complete, if anyone wants trustworthy data they're probably going to need to manage a Source (or better yet - as always - get globalnames to do that).

>this one is not found in the search 

If you mean the example search I provided, it explicitly excluded that. (With what I know, I can only provide an example that I think probably explains the system.)

**Comment by ccicero** [comment]
@campmlc how did you do the search? I searched MSB:Bird for CITES Appendix + Trochilidae (to narrow the search) and got 2,187 records including one Calliphlox amethystina (MSB:Bird:27936).

![image](https://github.com/user-attachments/assets/c1690a04-2c66-4563-9220-ccc1d5a2fc39)

**Comment by campmlc** [comment]
I just used @dustymc 's search replacing MVZ with MSB. If I remove Arctos legal and the = I can find these. I would definitely need an explanation as to why these results differ for the documentation. 
Preferably, I would submit a csv list of taxa to the Taxonomy module rather than in Search. Search results and individual records do not provide any info as to CITES level etc. Is this something we can add to the classification string in the record? It would be good to show it at the record level somewhere - e.g. a use restriction attribute that could be automagicked?

**Comment by mkoo** [comment]
ok, I have some ideas about this but first I have to ask if it's possible to have a record have more than one identification from two different taxonomies? (currently does not seem possible)

If possible, then you would have your primary identification from Arctos or WoRMS and then a second identification from Arctos Legal bearing the CITES I, II, III. That would function as a flag at the top of record next to the relevant info-- **_the taxon_**

**Comment by mkoo** [comment]
Or alternatively we do have search profiles with embedded saerch parameters like a saved search:
eg: https://arctos.database.museum/search.cfm?tax_rnk_1=CITES%20Appendix

**Comment by dustymc** [comment]
>possible to have a record have more than one identification from two different taxonomies

https://handbook.arctosdb.org/documentation/taxonomy.html#taxon-classification-sources

Yes, but not like that. Two IDs using two names which use two preferred sources without overlap would do that, but I think that's not what you're asking. 

https://handbook.arctosdb.org/documentation/taxonomy.html#taxonomic-concepts could be developed to accommodate this, but if you're asking because https://github.com/ArctosDB/arctos/issues/8348#issuecomment-2523835286 seems like a barrier then managing and using concepts for every record would probably seem a bit like a heavily armed and armored fortress standing in the way.

**Comment by dustymc** [comment]
Done?
